faunadb/request_response_info.py

```diff
@@ -0,0 +1,45 @@
+class RequestResponseInfo(object):
```

Comment:
Well, we could use them to provide more data to the exceptions so those can be more readily inspected (though inspecting via the observer is doable). Not sure if the exceptions as is allow for that though.